shared/modules/base_modules/integratable_module.go

```diff
@@ -0,0 +1,21 @@
+package base_modules
+
+import "github.com/pokt-network/pocket/shared/modules"
+
+var _ modules.IntegratableModule = &IntegratableModule{}
+
+type IntegratableModule struct {
```

Comment:
I don't think so. This struct is here to implement the interface that goes by the same name and I think it would be very confusing if it was named differently.
Module registration is an implementation detail. 

What is that makes you dislike the name and/or that's confusing?

Lemme add a comment anyway.

Reply:
💯